source/examples/time-series-forecasting-with-hpo/start_here.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>Can you use a <code>docref</code> directive here to show this is a reference to another documentation page?</p><p> </p><p>See this example https://github.com/rapidsai/deployment/blob/6c81beac81158ceae58ded3b273e8e4c440047cd/source/examples/rapids-azureml-hpo/notebook.ipynb?short_path=4cb4321#L27-L32 </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>